charts/fybrik/values.yaml

```diff
@@ -479,6 +479,106 @@ katalogConnector:
+    create: true
+    # Annotations to add to the service account
+    annotations: {}
+    # The name of the service account to use
+    name: openmetadata-connector
+
+  tls:
+    # MinVersion contains the minimum TLS version that is acceptable.
+    # If not provided, the system default value is used.
+    # Possible values are TLS-1.0, TLS-1.1, TLS-1.2 and TLS-1.3.
+    minVersion: TLS-1.3
+    # Specifies whether the openmetadata connector communication should use tls.
+    use_tls: false
+    # Specifies whether the openmetadata connector communication should use mutual tls.
+    use_mtls: false
+    certs:
+      # Name of kubernetes tls secret that holds the openmetadata-connector certificates.
+      # The secret should be of `kubernetes.io/tls` type.
+      # Relevant if tls is used.
+      # certSecretName: "test-tls-openmetadata-connector-certs"
+      certSecretName: ""
+      # Name of kubernetes secret that holds the certificate authority (CA) certificates
+      # which are used by openmetadata-connector to validate the connection to the manager
+      # if mtls is enabled.
+      # The CA certificates key in the secret should have `.crt` suffix.
+      # The provided certificates replaces the certificates in the system CA certificate store.
+      # If the secret is not provided then the CA certificates are taken from the system
+      # CA certificate store, for example `/etc/ssl/certs/`.
+      # cacertSecretName: "test-tls-ca-certs"
+      cacertSecretName: ""
+
+  podAnnotations: {}
+
+  # Pod Security Context. If set, the fields of podSecurityContext override
+  # the equivalent fields of .Values.global.podSecurityContext.
+  # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context
+  podSecurityContext: {}
+
+  # Container Security Context. If set, the fields of containerSecurityContext override
+  # the equivalent fields of .Values.global.containerSecurityContext.
+  # ref: https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#security-context-1
+  containerSecurityContext: {}
+
+  service:
+    type: ClusterIP
+    # For tls connection use port 8443
+    port: 8080
+
+  # Set the size limit of the data directory.
+  dataDirSizeLimit: 200Mi
+
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+    # limits:
+    #   cpu: 100m
+    #   memory: 128Mi
+    # requests:
+    #   cpu: 100m
+    #   memory: 128Mi
+
+  autoscaling:
+    enabled: false
+    minReplicas: 1
+    maxReplicas: 100
+    targetCPUUtilizationPercentage: 80
+    targetMemoryUtilizationPercentage:
+    # targetMemoryUtilizationPercentage: 80
+
+  vault:
```

Comment:
there is `vault` field in the values.yaml. you might want to use at least the address from it.

Reply:
When tried it, there was a problem. When I replace 
```
address: {{ .Values.openmetadataConnector.vault.address | default "http://vault.fybrik-system:8200"}}
```
with
```
address: {{ .Values.coordinator.vault.address | default "http://vault.fybrik-system:8200"}}
```
I find the following in the openmetadata-connector configuration file:
```
vault:
  address: http://vault.{{ .Release.Namespace }}:8200
```
Any thoughts? Thanks

Reply:
Use
`address: http://vault.{{ .Release.Namespace }}:8200` instead of `address: "http://vault.fybrik-system:8200"`